### PR TITLE
fix issue #1207, Windows: pkg run from disc other than C:

### PR DESCRIPTION
**Pull request by erossignon**

### CODE REVIEW

File: lib/common.ts [hunk 1]
```diff
@@ -119,8 +119,9 @@ function injectSnapshot(file: string) {
       // C:\
       file = file.slice(0, -1);
     }
-
-    return `${file[0]}:\\snapshot${file.slice(2)}`;
+    // by convention, on windows we use C:\\snapshot
+    const pathInSnapshot = `C:\\snapshot${file.slice(2)}`;
+    return pathInSnapshot;
```
Comment (jesec) [inline lib/common.ts:124]: return directly here?

lg2m in principle.

**Review by jesec — APPROVED** [review]

File: lib/common.ts (hunk 1 above)
Comment (erossignon) [reply lib/common.ts:124]: Yes, , I had a console.log at some point while developing, hence I store the intermediate result in a variable. 

Btw, I am struggling to find a way to create a non-regression test for this issue. as I am not sure we can easily create an alternative drive on the CI/CD Windows build the image.

Comment (jesec) [reply lib/common.ts:124]: You can mount a VHD.

Comment (erossignon) [reply lib/common.ts:124]: I found that `subst H: C:\MyFolder` will work.

File: lib/common.ts [hunk 2]
```diff
@@ -119,8 +119,8 @@ function injectSnapshot(file: string) {
       // C:\
       file = file.slice(0, -1);
     }
-
-    return `${file[0]}:\\snapshot${file.slice(2)}`;
+    // by convention, on windows we use C:\\snapshot
+     return  `C:\\snapshot${file.slice(2)}`;
```
Comment (jesec) [inline lib/common.ts:123]: extraneous whitespace here.

**Pull request merged**